cfme/tests/configure/test_access_control.py

```diff
@@ -3024,4 +3046,46 @@ def test_ansible_playbook_stdout():
             9.
             10. able to see standard output
     """
-    pass
+    user, group = custom_user
+    view = navigate_to(ansible_catalog_item, "SetOwnership")
+    view.form.select_an_owner.fill("Administrator")
+    view.form.select_group.fill(group.description)
+    view.submit.click()
+    ansible_catalog = appliance.collections.catalogs.create(
+        fauxfactory.gen_alphanumeric(),
+        description="my ansible catalog",
+        items=[ansible_catalog_item.name],
+    )
+    service_catalog = ServiceCatalogs(appliance, ansible_catalog, ansible_catalog_item.name)
+    ansible_service = MyService(appliance, ansible_catalog_item.name)
+
+    with user:
+        with appliance.context.use(ViaSSUI):
+            appliance.server.login(user)
```

Comment:
Since you're using `with user` you probably don't need to manually login.

Reply:
I tried but it was not working without this login()